packages/flutter/lib/src/gestures/constants.dart

```diff
@@ -95,3 +96,51 @@ const double kMaxFlingVelocity = 8000.0; // Logical pixels / second
 /// tap in a jump-tap gesture.
 // TODO(ianh): Implement jump-tap gestures.
 const Duration kJumpTapTimeout = Duration(milliseconds: 500);
+
+/// Like [kTouchSlop], but for mouse pointers.
+const double kMouseHitSlop = 1.0; // Logical pixels;
+
+/// Like [kPanSlop], but for mouse pointers.
+const double kPanMouseSlop = kMouseHitSlop * 2.0; // Logical pixels
+
+/// Like [kScaleSlop], but for mouse pointers.
+const double kScaleMouseSlop = kMouseHitSlop; // Logical pixels
+
+/// Determine the approriate hit slop pixels based on the [kind] of pointer.
+double computeHitSlop(PointerDeviceKind kind) {
+  switch (kind) {
+    case PointerDeviceKind.mouse:
+    case PointerDeviceKind.stylus:
```

Comment:
unsure if stylus should be treated like mouse or touch, probably the latter to be safe.